bash_scripting_101/bash_scripting_101.md

```diff
@@ -157,21 +172,21 @@ In our installation section, we covered how accessing CLI tools to run bash scri
 
```

Comment:
Can we give examples of these commands? This reads more like a reference list that goes at the end of the module, rather than a "here are these things for the first time" presentation.